package/crossplane.yaml

```diff
@@ -14,71 +14,7 @@ metadata:
-
-spec:
-  controller:
-    image: DOCKER_REGISTRY/provider-gcp-controller:VERSION
```

Comment:
@Feggah I think it would be worth including this update in the release notes, but it shouldn't have too much of an impact on users as they aren't actually constructing the `meta.pkg.crossplane.io` types 👍🏻